.bit/responses/1.4-Week 1 Step 4.md

```diff
@@ -15,17 +15,16 @@ name: Week 1 Step 4
 
 ### ✅  Tasks:
 - [ ]  Create a trigger that gets content from request parameters.
-- [ ] To move on, place your function URL in a [repository secret](https://docs.github.com/en/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository) called `HTTP_ENDPOINT` so we can check your function. Remember to commit the function's code in a file named `httptime.js` to the root of the `week1` branch!**
+- [ ]  Move on by:
```

Comment:
- [ ]  Place the...
- [ ] Switch....
- [ ] Commit...

This format would be great (sorry for not specifying more clearly the first time)